app/assets/scripts/map.js

```diff
@@ -19,9 +19,13 @@ function markerListener(marker, event) {
   var markerInfoWindowTemplate = $("#marker-info-window-template").html();
   var compiledMarkerInfoWindowTemplate =
     Handlebars.compile(markerInfoWindowTemplate);
+  if(marker.avg_rating == -1){
```

Comment:
i also think we can leave it to be -1 and check for -1 in the template to keep it consistent throughout (since it's set to -1 in the model). I'm curious why you chose to set it to 0?